Removed any cli-based Day 0 steps for getting started (#443)
128technology/docs: docs/intro_downloading_iso.md

## Introduction

> With your purchase of a SSR license, you will have obtained a set of credentials used to access the Session Smart Routing software. These credentials, in the form of a username and password will be used to access the software assets.

Reviewer: ```suggestion
With your purchase of a SSR license, you are provided a set of credentials used to access the Session Smart Routing software. These credentials, in the form of a username and password are used to access the software assets.
```
